### ISSUE TITLE
Cannot find module '../box-layout' with Electron 3.0 beta4

### ISSUE THREAD

**Issue by burtonator** [electron/spectron#317]
Not sure why this is happening but it's only being exposed on 3.0-beta4..

It's probably due to the RPC server

```
     unknown error: Cannot find module '../box-layout'
Error: Cannot find module '../box-layout'
    at Module._resolveFilename (internal/modules/cjs/loader.js:602:15)
    at Function.Module._resolveFilename (/home/burton/projects/polar-bookshelf/node_modules/electron/dist/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (internal/modules/cjs/loader.js:528:25)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.get [as BoxLayout] (/home/burton/projects/polar-bookshelf/node_modules/electron/dist/resources/electron.asar/browser/api/exports/electron.js:11:16)
    at EventEmitter.<anonymous> (/home/burton/projects/polar-bookshelf/node_modules/electron/dist/resources/electron.asar/browser/rpc-server.js:274:70)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (/home/burton/projects/polar-bookshelf/node_modules/electron/dist/resources/electron.asar/browser/api/web-contents.js:287:13)
    at WebContents.emit (events.js:182:13)
  Error: An unknown server-side error occurred while processing the command.
      at execute(<Function>, "require") - api.js:73:26

```

**Comment by remss** [comment]
Same thing with 3.0.0-beta.7

**Comment by burtonator** [comment]
Is there a reason this isn't fixed yet? How do people test their apps against beta 7 if spectron isn't working.  I'm using beta5 still but can't really test my code.

**Comment by MarshallOfSound** [comment]
This is fixed in latest master, the next 3.0 beta will contain the fix
